std/special/compiler_rt/comparetf2.zig

```diff
@@ -74,14 +74,12 @@ pub extern fn __getf2(a: f128, b: f128) c_int {
     if (aAbs > infRep or bAbs > infRep) return GE_UNORDERED;
     if ((aAbs | bAbs) == 0) return GE_EQUAL;
     // zig fmt issue here, see https://github.com/ziglang/zig/issues/2661
-    return if ((aInt & bInt) >= 0)
-        if (aInt < bInt)
-            GE_LESS
-        else if (aInt == bInt)
-            GE_EQUAL
-        else
-            GE_GREATER
-    else if (aInt > bInt)
+    return if ((aInt & bInt) >= 0) if (aInt < bInt)
+        GE_LESS
+    else if (aInt == bInt)
+        GE_EQUAL
+    else
+        GE_GREATER else if (aInt > bInt)
```

Comment:
I think this should be refactored.

Reply:
the formatting or the actual code?

Reply:
https://github.com/ziglang/zig/issues/2661

It's not really related to your change but trying to parse this hurts my brain.

Reply:
i'll put some `//zig fmt: off`/`//zig fmt: on` blocks around it.